pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -143,6 +143,7 @@ in stdenv.mkDerivation {
     "-Dsetfont-path=${kbd}/bin/setfont"
     "-Dtty-gid=3" # tty in NixOS has gid 3
     "-Ddebug-shell=${bashInteractive}/bin/bash"
+    "-Dglib=${stdenv.lib.boolToString (glib != null)}"
```

Comment:
What is the point in using `stdenv.lib` vs just `lib` here? There shouldn't be a difference at all and the latter is less code ;)

Reply:
I guess that is a pattern that is spread all over that file. We should fix that in a follow-up PR.